### ISSUE TITLE
Update: component.json to bower.json

### ISSUE THREAD

**Issue by paprikka** [yatskevich/grunt-bower-task#31]
Hi, 
I'm using new bower naming convention in my projects and I think it would be nice if:
- package list was stored in bower.json and not old component.json (which may cause conflicts with other package management tools)
- developer was able to change component.json filename from task options inside gruntfile, eg.: `bower.taskname.componentsJSON = 'customPackageFile.json'`

![image](https://f.cloud.github.com/assets/1511643/477906/fd3c1f8c-b7ec-11e2-9860-9fc26cd95a79.png)

What do you think about it?

**Comment by yatskevich** [comment]
This configuration option is supported by Bower itself. Please, take a look at Configuration section on [Bower's site](http://bower.io/)

**Comment by kvnwolf** [comment]
I agree with @paprikka, this task runner should be able to look for a defaul **bower.json** file.

Why? Because when i run `bower install [somepackage]`, it creates me a new **bower.json** file and keep **component.json**, so when i run again `grunt bower`, it will look in **component.json** instead of **bower.json**, ignoring all new packages.

**Comment by laurelnaiad** [comment]
I'm using this step in my build while this plugin conttinues to look for components.json:

```
  grunt.loadNpmTasks('grunt-contrib-copy');


    copy: {
      bowerToComponent: {
        files: {
          'component.json' : 'bower.json'
        }
      }
    },
```

Hopefully it will default to bower.json soon with a fallback to component.json.

**Comment by kvnwolf** [comment]
@stu-salsbury i don't think that would be the cleanest solution but still works. Let's wait for if @yatskevich adds it to the main source.

**Comment by laurelnaiad** [comment]
Right -- I didn't want to wait but I'm anxious to remove it.

**Comment by kvnwolf** [comment]
+1

**Comment by laurelnaiad** [comment]
In testing this again, it seems like all that is needed is to have
grunt-bower-task's package.json updated to 0.9.2 version of bower.

**Comment by kvnwolf** [comment]
Yes, that would be a solution, but we will stil having a 'custom' version of grunt-bower task and not the official.

Why i am so interested in the official task having the bower.json? Because i use grunt on the heroku side, so if i change it, my custom will be ignored by .gitignore and when grunt runs the task, it will look for the inexistent component.json file.

**Comment by laurelnaiad** [comment]
Oh I wasn't prodding _you_ to do it... :)

**Comment by yatskevich** [comment]
@paprikka, @kevin-wolf, @stu-salsbury fixed in v0.2.3

**Comment by kvnwolf** [comment]
w00t!
